spec/models/receptacle/downstream_aliquots_removal_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+RSpec.describe 'Receptacle::DownstreamAliquotsRemovalSpec::Mixin' do
+  describe '#allow_to_remove_downstream_aliquots?' do
+    # Labware
+    let(:original_plate) { create(:plate_with_untagged_wells, well_count: 1) }
+    let(:original_well) { original_plate.wells.first }
+    let(:plates) { create_list(:plate, 3, well_count: 1) }
+    let(:wells) { plates.map(&:wells).flatten }
+    let(:assets) { [original_well, wells].flatten }
```

Comment:
Can we try not to add more use of the word 'asset'? In this case, call it 'wells' or 'receptacles' instead? Because 'asset' is an old term, which was replaced by both labware and receptacle, so wherever it's still used I find it ambiguous.

Reply:
Yes, it is not a good name. Also just realised is not in use anywhere :( I'll remove it